src/backend/TrafficCourtsApi/citizen-api/Features/Lookups/LookupController.cs

```diff
@@ -7,31 +7,32 @@
 using System.Collections.Generic;
 using System.IO;
 using System.Text.Json;
+using System.Threading.Tasks;
 
 namespace Gov.CitizenApi.Features.Lookups
 {
-    [Route("api/[controller]/[action]")]
+    [Route("api/[controller]")]
     [ApiController]
     [OpenApiTag("Lookup API")]
     public class LookupController : Controller
     {
         private ILogger<LookupController> _logger;
-        private LookupsService _lookupsService;
-        public LookupController(ILogger<LookupController> logger, LookupsService lookupsService)
+        private ILookupsService _lookupsService;
+        public LookupController(ILogger<LookupController> logger, ILookupsService lookupsService)
         {
-            _logger = logger;
-            _lookupsService = lookupsService;
+            _logger = logger ?? throw new ArgumentNullException(nameof(logger));
+            _lookupsService = lookupsService ?? throw new ArgumentNullException(nameof(lookupsService));
         }
 
 
         [HttpGet]
         [Produces("application/json")]
-        [ProducesResponseType(typeof(ApiResultResponse<IEnumerable<Status>>), StatusCodes.Status200OK)]
+        [ProducesResponseType(typeof(ApiResultResponse<LookupsAll>), StatusCodes.Status200OK)]
         [ProducesResponseType(typeof(ApiMessageResponse), StatusCodes.Status500InternalServerError)]
-        public IActionResult CountCodes()
+        public async Task<IActionResult> Get()
         {
-            _logger.LogDebug("Get count codes info now.");
-            return Ok(_lookupsService.GetStatutes());
+            _logger.LogDebug("Get all Look Up Tables now.");
+            return Ok(new { result = await _lookupsService.GetAllLookUps() }); ;
```

Comment:
This does not return `ApiResultResponse<IEnumerable<Status>>` even though the returned object 'looks' like it.  This is not type safe.